app/config/config.php

```diff
@@ -146,6 +146,21 @@
     'schema_filter'  => '~^(?!'.MAUTIC_TABLE_PREFIX.'messenger_messages)~',
 ];
 
+if ($container->hasParameter('mautic.db_host_ro')) {
+    $dbalSettings['wrapper_class'] = \Mautic\CoreBundle\Doctrine\Connection\ConnectionMasterSlaveWrapper::class;
+    $dbalSettings['keep_slave']    = true;
+    $dbalSettings['slaves']        = [
+        'slave1' => [
+            'host'                  => '%mautic.db_host_ro%',
+            'port'                  => '%mautic.db_port%',
+            'dbname'                => '%mautic.db_name%',
+            'user'                  => '%mautic.db_user%',
+            'password'              => '%mautic.db_password%',
```

Comment:
Would be nice to have different connection settings for master / slave. Now we can change only the host.

Reply:
@patrykgruszka I do not know if we need different usernames and passwords, since the replica is usually using the same settings. Did you see a database cluster with different username and password?

Reply:
@mabumusa1 I personally have not seen such a configuration. I have thought about this and you are right, this should be enough.